tt_metal/distributed/mesh_device.cpp

```diff
@@ -441,4 +441,15 @@ size_t MeshDevice::num_program_cache_entries() const {
     return total_entries;
 }
 
+int MeshDevice::num_dram_channels() const {
+    return get_device_index(0)->num_dram_channels() * this->num_devices();
```

Comment:
Maybe we can have a private method `representative_device()` that returns `get_device_index(0)`? I saw several places in the codebase where we access the first device from a vector, would be great to make that assumption more explicit.

Reply:
done